src/client/client.ts

```diff
@@ -114,31 +108,18 @@ export class GraphQLTransportWSClient {
     return new Promise((resolve, reject) => {
       const handler = (error: Error[], result?: any) => {
         if (result) {
-          resolve({
-            data: result,
-            errors: error,
-          });
+          resolve(result);
         } else {
           reject(error);
         }
       };
 
-      this.executeRequest(options, handler, false);
+      this.executeRequest(options, handler);
```

Comment:
here we are subscribing to a result which we want need to cleanup eventually.
so either the handler should wait to completed event (let's add another argument to handle?) or query/mutation will call "unsubscribe" (need better name now) after getting 1 result.